### DISCUSSION TITLE
Convert glyph outlined path to single line bezier path #12

### DISCUSSION THREAD

**Discussion by alyfreym** [wieslawsoltes/Svg.Skia#132] [General]
Hi its possible using this lib convert outlined glyph path to single line path? I pin example image what I mean. Any suggestion please, or article.
![Group 3](https://user-images.githubusercontent.com/29518174/202890583-1350323c-bc2e-4dc0-970e-4e466a355767.png)

**Comment by wieslawsoltes** [comment]
Check https://github.com/wieslawsoltes/Core2D it does import Svg and show how to manipulate paths, it's not trivial.
